packages/@sanity/color-input/src/schema.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/display-name */
+import ColorInput from './ColorInput'
+import React from 'react'
+
+export default {
+  name: 'color',
+  type: 'object',
+  title: 'Color',
+  inputComponent: ColorInput,
+  fields: [
+    {
+      title: 'Hex',
+      name: 'hex',
+      type: 'string'
+    },
+    {
+      title: 'Alpha',
+      name: 'alpha',
+      type: 'number'
+    }
+  ],
```

Comment:
`hsl` seems to be missing.

Reply:
`hsl` is optional. Its discussed with @simen